.circleci/config.yml

```diff
@@ -863,6 +863,8 @@ jobs:
             brew install libsodium erlang@20
             brew link libsodium
             brew link --force erlang@20
+            brew install gmp erlang@20
```

Comment:
why not adding it to the install row above ? this looks like copy-paste nonsense to me 🤔